spec/features/applications/show_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "the application show" do
+  it "shows the application and all of its attributes" do
+    shelter = Shelter.create(name: "Mystery Building", city: "Irvine CA", foster_program: false, rank: 9)
+    pet = Pet.create(name: "Scooby", age: 2, breed: "Great Dane", adoptable: true, shelter_id: shelter.id)
+    application = Application.create(name: "John Dwayne", address: "1010 W 50th Ave, Denver, CO, 80020", description: "Background as a dog sitter", status: "Pending")
+
+    application.pets << pet
```

Comment:
I honestly had no idea this works, interesting! You can also just do `pet = application.pets.create(name: "Scooby"....etc)` and it should set up the relationship correctly there as well?

Admittedly haven't tried with the many-to-many setup, but that does work for one-to-many at least!